Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1203,12 +1203,9 @@ public string GetSmartApiUrl(string hostname, int? port = null)
         /// <inheritdoc/>
         public string GetLoopbackHttpApiUrl()
         {
-            if (NetManager.IsIP6Enabled)
-            {
-                return GetLocalApiUrl("::1", Uri.UriSchemeHttp, HttpPort);
-            }
-
-            return GetLocalApiUrl("127.0.0.1", Uri.UriSchemeHttp, HttpPort);
+            // Passing an external address cause GetBindInterface to return the externally facing interface on a multi-adapter system.
+            // LocalNetworkSubnets and LocalNetworkAddresses are used in conjunction with the ip address to help select the best interface.
+            return GetLocalApiUrl(NetManager.GetBindInterface("8.8.8.8", out var _), Uri.UriSchemeHttp, HttpPort);
```

Comment:
```suggestion
            return GetLocalApiUrl(NetManager.GetBindInterface("0.0.0.0", out var _), Uri.UriSchemeHttp, HttpPort);
```

I would prefer to not use Google's DNS server, even if it's never contacted.